src/hera/workflows/cron_workflow.py

```diff
@@ -1 +1,86 @@
-# TODO: Add cron workflow to v5
+from __future__ import annotations
+
+from types import ModuleType
+from typing import Optional
+
+from hera.workflows.models import (
+    CreateCronWorkflowRequest,
+    CronWorkflow as _ModelCronWorkflow,
+    CronWorkflowSpec,
+    CronWorkflowStatus,
+    LintCronWorkflowRequest,
+    ObjectMeta,
+)
+from hera.workflows.workflow import Workflow
+
+_yaml: Optional[ModuleType] = None
+try:
+    import yaml
+
+    _yaml = yaml
+except ImportError:
+    _yaml = None
+
+
+class CronWorkflow(Workflow):
+    concurrency_policy: Optional[str] = None
+    failed_jobs_history_limit: Optional[int] = None
+    schedule: str
+    starting_deadline_seconds: Optional[int] = None
+    successful_jobs_history_limit: Optional[int] = None
+    cron_suspend: Optional[bool] = None
+    timezone: Optional[str] = None
+    cron_status: Optional[CronWorkflowStatus] = None
```

Comment:
Note the `cron_` prefix here when the actual field value is without (as it would shadow the workflow spec's suspend and status)